tests/test_reverse_text_work_correctly.py

```diff
@@ -0,0 +1,89 @@
+import json
+from http import HTTPStatus
+import pytest
+from backend.main import lambda_reverse_text_backend
+
+
+def test_lambda_reverse_text_backend():
```

Comment:
Would be cool to parametrized these tests, given they are super similar to each other

Reply:
Done!